NAMESPACE

```diff
@@ -22,6 +23,7 @@ export(sce_to_anndata)
 export(sce_to_seurat)
 import(SingleCellExperiment)
 import(SummarizedExperiment)
+import(bluster)
```

Comment:
Same as the previous comment (I think roxygen/build will handle this automatically, but still)
```suggestion
```